examples/never-type/never_type.out

```diff
@@ -0,0 +1,6 @@
+# To run this sample, navigate to the directory that contains the
+# `.bal` file and issue the `ballerina run` command.
+$ ballerina run never_type.bal
+typedesc function (int) returns (never)
+typedesc xml<lang.xml:Element|lang.xml:Comment|lang.xml:ProcessingInstruction|lang.xml:Text>
+typedesc map<never>
```

Comment:
This is outdated right?

Also, is this going to be a BBE with no output?

Reply:
Yeah